prow/plugins/verify-owners/verify-owners_test.go

```diff
@@ -136,10 +152,31 @@ func IssueLabelsAddedContain(arr []string, str string) bool {
 	return false
 }
 
-func newFakeGitHubClient(changed []string, removed []string, pr int) *fakegithub.FakeClient {
+func ownersFilePatch(files []string, ownersFile string, nonEmptyPatch bool) map[string]string {
+	changes := emptyPatch(files)
+	if !nonEmptyPatch {
```

Comment:
Nit: Not inversing too much makes reading the code easier, so here instead of `!nonEmptyPatch` just `emptyPatch`

Reply:
Thank you for the review. Fixed.